base/docs/helpdb/Base.jl

```diff
@@ -7376,7 +7376,7 @@ for the effects of compilation.
     Array(dims)
 
 `Array{T}(dims)` constructs an uninitialized dense array with element type `T`. `dims` may
-be a tuple or a series of integer arguments. The syntax `Array(T, dims)` is also available,
+be a tuple or a series of integer arguments. The syntax `Array{T}(dims)` is also available,
```

Comment:
> (Please quote macros.... Apparently there are many creative usernames on github.....)

(Sorry, didn't notice the quote was stripped when quoting using the "r" shortcut )